code/style.css

```diff
@@ -1,4 +1,256 @@
 html {
-  background: #0025ff;
-  color: #fff;
+  max-width: 100%;
+}
+
+body {
+  background: #fffffc;
+  font-family: 'Merriweather', serif;
+  margin: 0 auto;
```

Comment:
`margin: 0` is completely enough, `auto` value will not change anything over here. It should be used inside `body`, when given element has only some % of total parent width